src/main/java/org/myrobotlab/service/meta/abstracts/AbstractSpeechSynthesisMeta.java

```diff
@@ -7,22 +7,10 @@
   private static final long serialVersionUID = 1L;
   public final static Logger log = LoggerFactory.getLogger(AbstractSpeechSynthesisMeta.class);
 
-  /**
-   * This class is contains all the meta data details of a service. It's peers,
-   * dependencies, and all other meta data related to the service.
-   * 
-   * @param name
-   *          n
-   * 
-   */
-  public AbstractSpeechSynthesisMeta(String name) {
-
-    super(name);
+  public AbstractSpeechSynthesisMeta() {
```

Comment:
this file probably shouldn't exist... abstract services shouldn't have metadata?

Reply:
there is shared implementation ...
currently all SpeechSynthesis service share a peer AudioFile
seems like configuration of wakeWord, mute, word substitute etc could be common vs copy/paste

Reply:
It should  marked as abstract!